packages/eslint-plugin/src/rules/test-id-consistent-naming.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const enforceKey = (context, validName) => jsxAttribute => {
+  context.report({
+    node: jsxAttribute,
+    message: `Test ID attribute should be named ${validName}.`,
+    fix(fixer) {
+      return fixer.replaceTextRange(jsxAttribute.name.range, validName);
+    },
+  });
+};
+
+module.exports = {
+  create(context) {
+    return {
+      'JSXAttribute[name.name = "testId"]': enforceKey(context, "data-test-id"),
```

Comment:
Do you seen any potential issues arising where `testId` is a valid React prop on a component? It's probably very unlikely that someone would use that name, but since it isn't a `data-*` prop it would be getting passed down into the component instead of turned into a html attribute.

Reply:
Not really, if you had `testId` and `data-test-id` on the same component I think _that_ would be confusing.